api/registrations/serializers.py

```diff
@@ -773,11 +773,8 @@ def _is_attached_file_valid(self, file_metadata, registering):
             return False
 
         # Confirm that the file has the expected name
-        normalized_file_names = [
-            normalize('NFD', file_metadata['file_name']),
-            normalize('NFC', file_metadata['file_name']),
-        ]
-        if attached_file.name not in normalized_file_names:
+        normalized_file_name = normalize_unicode_filename(file_metadata['file_name'])
```

Comment:
Nitpick:  if `normalize_unicode_filename` is returning a list, perhaps pluralize this to `normalize_unicode_filenames`.